### PR TITLE
[vtk] no absolute paths

### PR DESCRIPTION
**Pull request by autoantwort**
For #20469

### CODE REVIEW

File: ports/vtk/portfile.cmake [hunk 1]
```diff
@@ -296,6 +296,9 @@ file(REMOVE "${CURRENT_PACKAGES_DIR}/share/${PORT}/FindEXPAT.cmake")
 
 file(RENAME "${CURRENT_PACKAGES_DIR}/share/licenses" "${CURRENT_PACKAGES_DIR}/share/${PORT}/licenses")
 
+if(EXISTS "${CURRENT_PACKAGES_DIR}/include/vtk-9.0/vtkChemistryConfigure.h")
+    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/include/vtk-9.0/vtkChemistryConfigure.h" "${SOURCE_PATH}" "not/existing")
```
Comment (Cheney-W) [inline ports/vtk/portfile.cmake:300]: Could you show the content of the whole sentence in the vtkChemistryConfigure.h file after the replacement?

Comment (autoantwort) [reply ports/vtk/portfile.cmake:300]:
```
#ifndef vtkDomainsChemistryConfigure_h
#define vtkDomainsChemistryConfigure_h

/* Where the Blue Obelisk Data Repository element.xml file is found. */
#define VTK_BODR_DATA_PATH_BUILD "not/existing/Domains/Chemistry"

#endif
```

**Comment by strega-nil-ms** [comment]
Thanks!

**Comment by strega-nil-ms** [comment]
I forgot to click the merge button...

**Pull request merged**